### ISSUE TITLE
The background colour of the terminal seems to change for no reason since 0.43

### ISSUE THREAD

**Issue by cobrabr** [sedwards2009/extraterm#209]
![image](https://user-images.githubusercontent.com/724969/63313252-d2142600-c2d1-11e9-8976-0446d2c03357.png)

Notice the difference in colour between the border and the terminal itself.

_Originally posted by @cobrabr in https://github.com/sedwards2009/extraterm/issues/206#issuecomment-522828067_

**Comment by sedwards2009** [comment]
I've tracked this down to being a hardware acceleration related problem. The composition of the different elements on the screen is slightly incorrect for certain operating systems and GPUs/graphics cards. It is not just the background colour that is wrong. The whole terminal part is about 10% darker than it should be (on macOS).

**Comment by sedwards2009** [comment]
Fixed in v0.46.0.

**Comment by cobrabr** [comment]
Still happening for me:
![image](https://user-images.githubusercontent.com/724969/66683511-a2d0c400-ec45-11e9-9c6e-35af6510f40b.png)

**Comment by sedwards2009** [comment]
Is the "reduce graphics" option turned on in the Settings?

**Comment by sedwards2009** [comment]
Also, what GPU chip are you using? I could reproduce this problem on macOS with some old-ish Intel integrated GPU, but I don't have a Windows machine which does the same.

**Comment by cobrabr** [comment]
I've tried it with both it on and off, with the same results. My GPU is a GeForce RTX 2070.

I also noticed that, while in the Settings tab, if I hover the mouse over the tab title and the tab itself, the background color flickers between the right colour and the darker one there as well: 

![2019-10-12_18-26-07](https://user-images.githubusercontent.com/724969/66708409-09c1ac00-ed1e-11e9-80e8-3b8dc438d944.gif)

**Comment by cobrabr** [comment]
I don't know if this helps, but I noticed that the effect is different depending on the theme. If I choose "xterm", for example, this is what I get in Settings > Appearance:

![image](https://user-images.githubusercontent.com/724969/66708460-b7cd5600-ed1e-11e9-8648-d006a02f35db.png)

And this is what the actual terminal looks like:

![image](https://user-images.githubusercontent.com/724969/66708472-dc293280-ed1e-11e9-9651-9006f7e6eb46.png)

However, the flicker that I mentioned on my previous post looks different:

![2019-10-12_18-34-45](https://user-images.githubusercontent.com/724969/66708482-01b63c00-ed1f-11e9-8aca-759c4096f692.gif)

**Comment by sedwards2009** [comment]
My first advice is to keep your graphics driver up to date. Updating the Electron version at my end *may* fix the problem or it may not. One thing which probably will work is to add an option to disable hardware acceleration. I don't really want people getting a slow Extraterm experience but it looks like the option is still needed in your case @cobrabr .

**Comment by cobrabr** [comment]
I have the latest NVidia driver, so we're good there. It's weird because it used to work before, and the GPU was the same back then. I could try installing the latest version at work, where the hardware is completely different to see if I get different results, but I don't know if that'll help.

**Comment by sedwards2009** [comment]
Option to disable HW acceleration added in v0.47.0.

**Comment by cobrabr** [comment]
Looks like disabling HW acceleration did the trick. However, after turning it off and restarting ExtraTerm, whenever I open the Settings tab, HW acceleration is checked on and asking for a restart (even though I didn't check the checkbox on myself):

![image](https://user-images.githubusercontent.com/724969/69013766-c7f5d800-0951-11ea-92c5-cd8760c83b8a.png)

If I restart it again, HW is on and the problem comes back (at which point I have to turn it back off and restart one more time). 

TL;DR, HW acceleration turns itself back on every other restart and needs to be manually turned off again.

**Comment by sedwards2009** [comment]
I can reproduce this problem.

**Comment by sedwards2009** [comment]
The settings checkbox is fixed in 0.48.0.
